mu-plugins/10up-plugin/composer.json

```diff
@@ -9,9 +9,15 @@
       "email": "info@10up.com"
     }
   ],
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/10up/wp-framework.git"
+    }
+  ],
   "require": {
     "php": ">=8.3",
-    "haydenpierce/class-finder": "^0.5.3"
+    "10up/wp-framework": "dev-trunk"
```

Comment:
can we use tagged versions instead? 🤔

Reply:
(I realize that means making the wp-framework repo public but that is needed anyways. This should be part of the OSP version of the scaffold)

Reply:
Yup, this in in the todo of the PR, I just wanted to get some eyes on it internally before making the repo public and tagging 🙂